package.json

```diff
@@ -5,23 +5,17 @@
     "main": "index.js",
     "type": "module",
     "scripts": {
-        "test-old": "cross-env CI=true jest --collect-coverage --coverage --watch=all --coverageDirectory=coverage/unit ./cdc-accelerator",
-        "test": "cross-env CI=true npm run update-sap-cdc-toolkit && jest --collect-coverage --coverage --watch=all --coverageDirectory=coverage/unit ./cdc-accelerator/feature",
+        "test": "cross-env CI=true npm run update-sap-cdc-toolkit && jest --collect-coverage --coverage --watch=all --coverageDirectory=coverage/unit ./cdc-accelerator/core ./cdc-accelerator/feature",
```

Comment:
shouldn't be "cross-env CI=true npm run update-sap-cdc-toolkit && jest --collect-coverage --coverage --watch=all --coverageDirectory=coverage/unit ./cdc-accelerator" ?

Additionally, the files **/test.gigyaResponse*.js should also be on tests directory
Also, the files in directory services/gigya are being used?

Reply:
If we change the test script in that way, the tests in cdc-accelerator/script will also be executed. Do we want that since we now have them in their own script?

Reply:
No, you are right. Ignore my comment